upgrade_downgrade/test_dual_control_plane_upgrade_downgrade.sh

```diff
@@ -22,7 +22,7 @@ ROOT=$(dirname "$WD")
 
 ISTIO_NAMESPACE="istio-system"
 # Maximum % of 503 response that cannot exceed
-MAX_503_PCT_FOR_PASS="15"
+MAX_503_PCT_FOR_PASS="5"
```

Comment:
this percentage initially set for normal upgrade of istio testing https://github.com/istio/tools/blob/master/upgrade_downgrade/test_upgrade_downgrade.sh#L67
yeah, i agree, this might be too large to capture the useful signal

Reply:
ah I see, so we need a higher threshold when doing in-place upgrade, makes sense. we should still be able to decrease the dual control plane test's 503 threshold as done in this PR though, right?

Reply:
@ostromart needs your input here

Reply:
offline discussed, we could try this first.